### DISCUSSION TITLE
不同订阅之间的区别是什么 ？

### DISCUSSION THREAD

**Discussion by yu-steven** [yu-steven/openit#62] [Q&A] [answered]
如下 ⬇️

**Comment by myotsuki** [comment]
请别污染社区谢谢。No politics。

**Reply by yu-steven** [reply]
> 请别污染社区谢谢。No politics。

所以你有要问的问题吗？

**Reply by Vogadero** [reply]
> 请别污染社区谢谢。No politics。

何出此言？和政治啥关系？？？

**Reply by yu-steven** [reply]
> > 请别污染社区谢谢。No politics。
> 
> 何出此言？和政治啥关系？？？

人也没说政治哈哈哈哈

**Comment by pipi233332333** [comment]
冒泡

**Reply by yu-steven** [reply]
🌚

**Comment by zhabgzilongdhdd** [comment]
全局和代理有什么区别

**Reply by yu-steven** [reply]
一个意思，关键得看什么语境

**Comment by Vogadero** [comment, marked as answer]
这个怎么用的？开头那通用和专用区别是啥？把链接复制到软件就能自动更新服务器的地址下来吗？

**Reply by yu-steven** [reply]
**不同订阅之间的区别**
Base64 订阅可在大多数翻墙软件中使用
除了以下这三个软件
小火箭 订阅可在Shadowrocket中使用 
Clash 订阅可在Clash系列软件中使用 
Quanx 订阅可在Quantumult X中使用

是的，如您使用其他订阅一样，把链接复制到软件并选择自动更新订阅即可食用
